src/components/families/FamilyDetailsSidebar.tsx

```diff
@@ -19,6 +19,13 @@ const useStyles = makeStyles(() => ({
   pb: {
     paddingBottom: 16,
   },
+  heading: {
+    color: "#42526E",
+    fontFamily: "Roboto",
+    fontStyle: "normal",
```

Comment:
do we need to specify these? just wondering :0

Reply:
I think we're good to remove those